{"date":"Thu Dec 20 2018 10:54:39 GMT+0100 (W. Central Africa Standard Time)","process":{"pid":2000,"uid":null,"gid":null,"cwd":"C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v10.3.0","argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\app.js"],"memoryUsage":{"rss":37298176,"heapTotal":21815296,"heapUsed":12598008,"external":17903995}},"os":{"loadavg":[0,0,0],"uptime":9524},"trace":[{"column":16,"file":"C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\app.js","function":null,"line":432,"method":null,"native":false},{"column":14,"file":"C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\utils.js","function":"err","line":415,"method":null,"native":false},{"column":25,"file":"C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\utils.js","function":"executeCallback","line":404,"method":null,"native":false},{"column":21,"file":"C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\mongo_client.js","function":null,"line":271,"method":null,"native":false},{"column":5,"file":"C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\mongo_client.js","function":"connectCallback","line":946,"method":null,"native":false},{"column":11,"file":"C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\mongo_client.js","function":null,"line":791,"method":null,"native":false},{"column":11,"file":"internal/process/next_tick.js","function":"process._tickCallback","line":61,"method":"_tickCallback","native":false}],"stack":["AssertionError [ERR_ASSERTION]: null == { MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoNetworkError: connect ECONNREFUSED ...","    at C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\app.js:432:16","    at err (C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\utils.js:415:14)","    at executeCallback (C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\utils.js:404:25)","    at C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\mongo_client.js:271:21","    at connectCallback (C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\mongo_client.js:946:5)","    at C:\\Users\\Mobolaji.Aina\\Documents\\nodejs\\kayxlavcomms\\node_modules\\mongodb\\lib\\mongo_client.js:791:11","    at process._tickCallback (internal/process/next_tick.js:61:11)"],"level":"error","message":"uncaughtException: null == { MongoNetworkError: failed to connect to server [127.0.0.1:27017] on first connect [MongoNetworkError: connect ECONNREFUSED ...","timestamp":"2018-12-20T09:54:39.175Z"}
